mappings/net/minecraft/client/gui/widgets/Button.mapping

```diff
@@ -19,9 +19,20 @@ CLASS ke net/minecraft/client/gui/widgets/Button
 		ARG 2 x
 		ARG 3 y
 		ARG 4 text
+	METHOD a mouseReleased (II)V
+		ARG 1 mouseX
+		ARG 2 mouseY
 	METHOD a render (Lnet/minecraft/client/Minecraft;II)V
+		ARG 1 minecraft
+		ARG 2 mouseX
+		ARG 3 mouseY
+	METHOD a getYImage (Z)I
```

Comment:
I have no idea what the "y image" is meant to mean

Reply:
dw me neither

but it is from mojang names so this is good